core/app/models/spree/store_credit.rb

```
@@ -184,7 +185,9 @@ def generate_authorization_code
 
     class << self
       def default_created_by
-        Spree.user_class.find_by(email: DEFAULT_CREATED_BY_EMAIL)
+        Spree::Deprecation.warn('StoreCredit#default_created_by is deprecated and will be removed in Spree 6.0. Please use store.users.first instead.')
```

RuboCop (1.75.5)

[convention] Line is too long. [151/150]

(Layout/LineLength)